src/data.js

```diff
@@ -1,33 +1,147 @@
-const listArray = [
-  {
-    description: 'Complete To-Do list',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Take a walk',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Watch a movie',
-    completed: false,
-    index: 2,
-  },
-];
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] Since `linter` are there to help us implement best practices, instead of disabling `eslint`,  please kindly fix the issue.

Reply:
- [ ] Calling the function many times before declaring;  I also mentioned this issue on my pull request. It is difficult for me to arrange every code again with the time I have. If you give me a tip, I will implement it in a future project.
![Linters](https://user-images.githubusercontent.com/65129228/144715577-38a5eb22-7a59-4226-9bc7-81b96bff98be.png)
.